client/layout/store-alerts/test/index.js

```diff
@@ -13,11 +13,13 @@ const alerts = [
 	{
 		title: 'Alert title 1',
 		content: 'Alert content 1',
+		status: 'unactioned',
```

Comment:
Although we filtered in the `useSelect` by `unactioned` we didn't pass `unactioned` alerts into our tests. This fixes the tests.